commands/build.sh

```diff
@@ -35,8 +35,24 @@ get_caches_for_service() {
   fi
 }
 
+get_caches_to_service() {
+  local service="$1"
+
+  # Read any cache-to parameters provided
+  for line in $(plugin_read_list CACHE_TO) ; do
+    IFS=':' read -r -a tokens <<< "$line"
+    service_name=${tokens[0]}
+    service_image=$(IFS=':'; echo "${tokens[*]:1}")
+
+    if [ "${service_name}" == "${service}" ]; then
+      echo "$service_image"
+    fi
+  done
+}
```

Comment:
I believe it makes sense to have this new behaviour respect the `no-cache` option... and, as with the other it may make sense to unify it with the `get_caches_from_service` so that it has an additional parameter of what option to look for thus avoiding two functions that do exactly the same

Reply:
@toote on fence on this one, as get where you are coming from but `--no-cache` is documented for not using cache _during_ the Building of an Image, where as `cache_to` is for specifying where cache should be exported to _after_ the Building of an Image. 

So feeling it wouldn't make sense to use `no-cache` to also block exporting cache, else if not wanting to export cache then `cache-to` values shouldn't be specified? 

https://docs.docker.com/reference/cli/docker/compose/build/
https://docs.docker.com/reference/compose-file/build/#cache_to

Reply:
though in retrospec it might make sense that `get_caches_for_service` is also not effected by `no-cache` parameter given itself configures the `--no-cache` flag itself for the `docker compose` commands which would ignore any `cache_from` values in the Docker Compose file